styles/colors-game.less

```diff
@@ -30,9 +30,9 @@
 
 // Alert
 // ---------------------------------------
-@sand-alert-overlay-text-color-rgb: var(--sand-alert-overlay-text-color-rgb, 255, 255, 255); // #ffffff
-@sand-alert-overlay-bg-color-rgb: var(--sand-alert-overlay-bg-color-rgb, 0, 0, 0); // #000000
-@sand-alert-overlay-bg-color-opacity: 75%;
+@sand-alert-overlay-bg-color: var(--sand-alert-overlay-bg-color, rgb(var(--sand-alert-overlay-bg-color-rgb, 0, 0, 0))); // #000000
```

Comment:
What is `--sand-alert-overlay-bg-color-rgb`?

Reply:
It is a kind of fallback for old legacy uses. I will update the PR description.

Reply:
Updated